src/routes/account.js

```diff
@@ -152,12 +152,37 @@ router.get('/logout', async (_request, response) => {
 });
 
 router.get('/forgot-password', async (request, response) => {
-	response.render('account/forgot-password');
+    const renderData = {
```

Comment:
I noticed that there are inconsistent tabs and spaces in this file. The Pretendo projects use tabs for indentation as a standard.

Reply:
Thanks for showing me that! Something happened while I was commiting and the indentation got messed up.

Reply:
That commit didn't fix the issue.